README.md

```diff
@@ -1,2 +1,25 @@
-# advisory-group
-Enterprise Advisory Group
+# Enterprise Advisory Group
+
+The Enterprise Advisory Group consists of corporate representatives in leadership roles responsible for managing for large-scale deployments of Node.js in an Enterprise environment. We strive to provide timely feedback to the [TSC](https://github.com/nodejs/TSC), the [CommComm](https://github.com/nodejs/community-committee), the [Node.js project](https://github.com/nodejs/node) and the Node.js Foundation about the needs of Node.js Enterprise users.
+
+Thank you for your interest and support. Please [create an issue](https://github.com/nodejs/user-feedback/issues) in this repo if you are interested in dedicating 5+ hours a month during the formation process.
+
+### Governance and Contributing
+The Enterprise Advisory Group has adopted the core governance and contributing policies of Node.js. This group is a part of the [Node.js End User Feedback Inititative](https://github.com/nodejs/user-feedback) which is under the charter of the [Node.js Community Committee (CommComm)](https://github.com/nodejs/community-committee).
+
+Members are are expected to provide feedback on how proposed changes in the Node.js project will affect Enterprise teams, business goals, and application performance at scale. Members attend regular group meetings to provide feedback and respond to occasional outreach requests. 
```

Comment:
Typo: `are are`

The whole "are expected to" phrasing is should go in my opinion. `Members provide feedback...` instead of `Members are expected to provide feedback...`

Reply:
@trott Hmm, I'm not 100% comfortable with your proposed change to remove "are expected to". Though, I'm happy to clarify the statement.

The reason I use "expected to" here is to 1) establish expectations amongst members and potential members 2) avoid implying that this group is in any way a gate or has any authority to block actions of the TSC, or any other area of the Node.js project.

Can you help rephrase the statement to better convey that?

Reply:
My suggestion was just that: A suggestion. If you think the "are expected to" part is important, then by all means, leave it in. 

Do fix the `are are` though. I'm pretty adamant about that one. :-D

Reply:
You bet. Fixed. (I didn't want to collapse this section until I had addressed your concern.)